test/pylib/scylla_cluster.py

```diff
@@ -1524,7 +1526,8 @@ async def _after_test(self, _request) -> dict[str, bool]:
             self.cluster.after_test(self.current_test_case_full_name, success)
         finally:
             logging.getLogger().removeHandler(self.test_case_log_fh)
-            pathlib.Path(self.test_case_log_fh.baseFilename).unlink()
+            if success:
```

Comment:
The commit message does not explain this change. It seems you changed the code to in some case (which?) when something (?) didn't succeed don't delete the log file. What does this change, and why?

Reply:
This is just to not delete scylla manager log file in case the test failed.

Reply:
Ok, I guess. But since this was always deleted and nobody ever complained, why do we suddenly need it? :-)
But I'll merge. Thanks for updating the commit message.

Reply:
I decided it's a good idea to unify the behavior about not deleting the log files for failed tests and as I'm already messing around with these specific logs why not to add this small enhancement?